test/test_distributions.py

```diff
@@ -333,6 +341,31 @@ def get_sp_dist(jax_dist):
     T(dist.VonMises, 2.0, 10.0),
     T(dist.VonMises, 2.0, jnp.array([150.0, 10.0])),
     T(dist.VonMises, jnp.array([1 / 3 * jnp.pi, -1.0]), jnp.array([20.0, 30.0])),
+    T(
+        SineBivariateVonMises,
+        jnp.array([0.0]),
+        jnp.array([0.0]),
+        jnp.array([5.0]),
+        jnp.array([6.0]),
+        jnp.array([2.0]),
```

Comment:
Is it required that those are at least 1D?

Reply:
Yes, however, with your suggested changes, this may be resolved. I'll check and update the test cases appropriately.

Reply:
If this is required, you should change the `arg_constraints` in the distribution appropriately (using `independent`). But from the implementation, it seems to be that those parameters are scalars (modulo batch_shape).

Reply:
It looks like the scalars work with your latest suggested changes.

Reply:
Could you add one test that invokes scalars?

Reply:
Sure, I’ll update once I back from holidays next week (forgot my laptop C:).